### ISSUE TITLE
Full Accessibility & 508 Office Audit [v2 VA Facility Selection page]

### ISSUE THREAD

**Issue by lenaecb** [department-of-veterans-affairs/va.gov-team#15308]
## Artifacts
- Access info: use Judy.Morrison
- Link to your accessibility test plan:
  - Accessibility check list ticket - [here](https://app.zenhub.com/workspaces/vft-59c95ae5fda7577a9b3184f8/issues/department-of-veterans-affairs/va.gov-team/14169)

## Additional Notes
- Currently this new v2 page is on staging and being tested with some of VAOS users
- the change is that we are now showing users a flat facility list (vs. previous version where user had to make a parent then child facility selection)
- when a user has a 'home address' on file - we are sorting facilities returned based on home address
- if user chooses to use 'use my location' - list is sorted based on user's browsers location
- if user doesn't have home address on file - user gets facility list in alphabetical order

**Comment by noahgelman** [comment]
Audit for this ticket has been completed.

**Comment by laurenernest** [comment]
All 4 issues have been resolved.
